WordPress/Classes/ViewRelated/System/Floating Create Button/CreateButtonCoordinator.swift

```diff
@@ -24,8 +24,7 @@ import WordPressUI
     private let noticeAnimator = NoticeAnimator(duration: 0.5, springDampening: 0.7, springVelocity: 0.0)
 
     private func notice(for blog: Blog) -> Notice {
-        let showsStories = blog.supports(.stories)
-        let title = showsStories ? NSLocalizedString("Create a post, page, or story", comment: "The tooltip title for the Floating Create Button") : NSLocalizedString("Creates new post, or page", comment: " Accessibility hint for create floating action button")
+        let title = NSLocalizedString("Creates new post, or page", comment: " Accessibility hint for create floating action button")
```

Comment:
I wonder if we should use the same copy as showCreateButton - "Create a post or page"

Reply:
Good point – I moved this to `Stings` too.